packages/strapi-helper-plugin/lib/src/utils/auth.js

```diff
@@ -24,11 +24,14 @@ const auth = {
       const videos = auth.get('videos');
       const onboarding = auth.get('onboarding');
       const strapiUpdateNotification = auth.get('STRAPI_UPDATE_NOTIF');
+      const localeLang = localStorage.getItem('strapi-admin-language');
 
       localStorage.clear();
+
       localStorage.setItem('videos', JSON.stringify(videos));
       localStorage.setItem('onboarding', onboarding);
       localStorage.setItem('STRAPI_UPDATE_NOTIF', strapiUpdateNotification);
+      localStorage.setItem('strapi-admin-language', localeLang);
```

Comment:
ℹ️ This restores the key in local storage when the user logouts. This allows keeping the language selected by the user even if they are not connected